docs/features/multiple-networks/multi-homing.md

```diff
@@ -138,7 +210,7 @@ spec:
   config: |2
     {
             "cniVersion": "1.0.0",
-            "name": "localnet-network",
+            "name": "localnet-physical-network",
```

Comment:
these are kind of redundant but I won't fight for changing it.

Reply:
change to the tenant<color> format